### PR TITLE
Adding Citrix Ingress Controller charts

### PR DESCRIPTION
**Pull request by apoorva-05**

### CODE REVIEW

File: charts/citrix-k8s-cpx-ingress-controller/v1.6.1/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: "1.6.1"
+description: A Helm chart for Citrix ADC CPX with Citrix ingress Controller running as sidecar.
+name: citrix-k8s-cpx-ingress-controller
+version: 1.6.1
+icon: https://raw.githubusercontent.com/citrix/citrix-helm-charts/gh-pages/icon.png
```
Comment (guangbochen) [inline charts/citrix-k8s-cpx-ingress-controller/v1.6.1/Chart.yaml]: can we please have the icon in the parent directory and refer it as `file://` so users with the air-gapped environment can also view it properly.  please refer to [this](https://github.com/rancher/charts/blob/master/charts/wordpress/v7.3.8/Chart.yaml#L6) as example, thanks.

File: charts/citrix-k8s-cpx-ingress-controller/v1.6.1/questions.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
+questions:
```
Comment (guangbochen) [inline charts/citrix-k8s-cpx-ingress-controller/v1.6.1/questions.yml]: can u please add the partner label as below, thanks.
```
labels:
  io.rancher.certified: partner
```

**Comment by apoorva-05** [comment]
> 
> 
> Can u please help to squash the above commits into 1, thanks.



> 
> 
> Can u please help to squash the above commits into 1, thanks.

I tried to squash it to 1 and was facing some issues, but have reduced it to three commits.

> 
> 
> Can u please help to squash the above commits into 1, thanks.

Done. Sorry for the inconvenience.

**Comment by guangbochen** [comment]
Thanks, LGTM.

**Pull request merged**